### PR TITLE
kubernetes v1.2

### PR DESCRIPTION
**Pull request by pbx0**
I removed the testing of the v1.1.7 and v1.1.8 image. I see no reason to keep those around and gate OS releases on them. Also, by updating the setup scripts to v1.2 those images will no longer work so its not worth the maintenance burden.

### CODE REVIEW

**Comment by crawford** [comment]
Can you rebase those last two patches?

File: kola/tests/kubernetes/basic.go [hunk 1]
```diff
@@ -20,10 +20,56 @@ import (
...
+// register a separate test for each version tag
+var basicTags = []string{
+	"v1.1.7_coreos.2",
+	"v1.1.8_coreos.0",
+}
+
+func init() {
+	for i := range basicTags {
+		// use closure to store a version tag in a Test
+		t := basicTags[i]
```
Comment (crawford) [inline kola/tests/kubernetes/basic.go]: I hate this language.

**Comment by crawford** [comment]
Otherwise, LGTM.

**Comment by pbx0** [comment]
rebased and made some kubernetes specific changes to the install script.

**Comment by crawford** [comment]
LGTM

**Pull request merged**